### PR TITLE
added allure to new user test

### PR DESCRIPTION
**Pull request by maxreg**

### CODE REVIEW

**Review by Goriander — DISMISSED** [review]

**Review by ArtemHurin — DISMISSED** [review]

**Review by SET-SKYNET — APPROVED** [review]

**Review by SET-SKYNET — APPROVED** [review]

**Review by SET-SKYNET — DISMISSED** [review]

File: src/main/java/test/TestMaksym/TestNewUser.java [hunk 1]
```diff
@@ -1,57 +1,55 @@
...
+
+import static org.testng.Assert.assertEquals;
+@Listeners (test.class)
+
+public class TestNewUser extends test {
+  
+  @Severity(SeverityLevel.CRITICAL)
+  @Description("Verify creation of the new user account")
+	@Test
+	public void testCreateNewUser() throws InterruptedException {
```
Comment (ostret) [inline src/main/java/test/TestMaksym/TestNewUser.java:24]: fix identation

File: src/main/java/test/TestMaksym/TestNewUser.java [hunk 2]
```diff
@@ -1,57 +1,55 @@
...
+		UserRepository userRepository = new UserRepository();
+
+		MainPage mainPageUserUnregistered = new MainPage();
+		Thread.sleep(500); // for demonstration purposes only
+
+		// click at sign in button to get new LoginPage
+		LoginPage loginPage = mainPageUserUnregistered.clickSignIn();
+		Thread.sleep(500); // for demonstration purposes only
+
+		// click at link "No account? Create one here" to get new RegisterPage
```
Comment (ostret) [inline src/main/java/test/TestMaksym/TestNewUser.java:35]: love the comments
you shouls also add them as @step annotations

**Review by SET-SKYNET — APPROVED** [review]

File: src/main/java/test/TestMaksym/TestNewUser.java (hunk 1 above)
Comment (maxreg) [reply src/main/java/test/TestMaksym/TestNewUser.java:24]: tried to fix, commit via --amend and pushed,   but no changes appeared in this merge
if you don't mind let us merge this 'as is' and I will provide changes the next time

File: src/main/java/test/TestMaksym/TestNewUser.java (hunk 2 above)
Comment (maxreg) [reply src/main/java/test/TestMaksym/TestNewUser.java:35]: same as my comment above

**Review by Oleg-Decik — APPROVED** [review]

**Pull request merged**